common/lib/xmodule/xmodule/xml_module.py

```diff
@@ -428,6 +429,13 @@ def add_xml_to_node(self, node):
         """
         # Get the definition
         xml_object = self.definition_to_xml(self.runtime.export_fs)
+        for aside in self.runtime.get_asides(self):
+            if aside.needs_serialization():
+                aside_node = etree.Element("unknown_root", nsmap=XML_NAMESPACES)
+                aside.has_children = False
```

Comment:
@cpennington I've created the related PR to fix this bug: https://github.com/edx/XBlock/pull/337